agent/internal/fluent.go

```diff
@@ -292,6 +305,12 @@ func startLoggingContainer(
 				Memory:   1 << 30,
 				NanoCPUs: 1000000000,
 			},
+			LogConfig: container.LogConfig{
+				Type: "json-file",
+				Config: map[string]string{
+					"max-size": "10m",
```

Comment:
this is unrelated, just so the fluentbit container itself will do log rotation since just it's `INFO` flush logs log about a 1kb/s since we're flushing every .05 seconds.. it would take like 2 months to be 2gb of log space but still 🤷